### PR TITLE
Feat/web 26

### PR DESCRIPTION
**Pull request by Dierish05**
## Description
- este pr trae la seccion del who we are

## Pull request checklist
Please check if your PR fulfills the following requirements:
- [X] My code follows the code style of this project.
- [ ] I have added tests to cover my changes.
- [ ] All new and existing tests passed. (Add screenshot)
- [ ] Is my code avoiding performance problems (ex: rendering issues, amount of queries against the database, n+1 problem, infinite loops, memory usage)
- [ ] Is my code handling integration exceptions (ex: timeout error while calling an external service)
- [ ] My code includes usage metrics (ex: google analytics events, logs)
- [ ] My code includes restriction of access based on roles and permissions

## Pull request type
 
 

Please check the type of change your PR introduces:
- [ ] Bugfix (non-breaking change which fixes an issue)
- [X] Feature (non-breaking change which adds functionality)
- [ ] Code style update (formatting, renaming)
- [ ] Refactoring (no functional changes, no api changes)
- [ ] Build related changes
- [ ] Documentation content changes
- [ ] Other (please describe): 

## Does this introduce a breaking change?

- [ ] Yes
- [X] No

## JIRA Ticket Link

JIRA Ticket: https://kodifydevs.atlassian.net/browse/WEB-26

## Commands to be executed or config files to be changed
- 

## Other information / Screenshots (Required for UI Changes)

image
image
image
image

### CODE REVIEW

File: apps/web/containers/index.js [hunk 1]
```diff
@@ -0,0 +1,2 @@
+export {default as HowWeAre} from './src/HowWeAre';
+//export {default as Hero} from './src/Hero';
```
Comment (HamiltonG98) [inline apps/web/containers/index.js:2]: Elimina este comentario

File: apps/web/containers/src/Hero/index.js [hunk 2]
```diff
@@ -0,0 +1 @@
+//export { default } from './Hero';
```
Comment (HamiltonG98) [inline apps/web/containers/src/Hero/index.js:1]: Por que esta comentado?

File: apps/web/containers/src/HowWeAre/HowWeAre.jsx [hunk 3]
```diff
@@ -0,0 +1,69 @@
+import React from 'react'
+import { Button, Container } from "@packages/components";
+
+function HowWeAre() {
+  return (
+    <section>
+      {/* hacer supongo un grid o flex para dejar lista la parte de desktop */}
```
Comment (HamiltonG98) [inline apps/web/containers/src/HowWeAre/HowWeAre.jsx:7]: Podes actualizar este comentario ahora que ya esta listo

File: apps/web/containers/src/HowWeAre/HowWeAre.jsx [hunk 4]
```diff
@@ -0,0 +1,69 @@
...
+      <div className=" w-full bg-rich-black relative">
+        <Container>
+          <div className=" relative">
+            <div className=" absolute top-[49px] left-[22px] hidden lg:block">
+              <img src="Group23.svg"/>
+            </div>
+            <div className=" lg:h-[392px] lg:flex lg:mx-auto">
+              <div className=" flex w-fit pt-6 lg:pt-0 mb-12 mx-auto lg:my-auto" >
```
Comment (HamiltonG98) [inline apps/web/containers/src/HowWeAre/HowWeAre.jsx:15]: Las clases deben llevar un orden, todas las clases de mobile van primero, despues todo lo de tablet y por ultimo todo lo de desktop. Hay un lg: en medio de las de mobile

File: apps/web/containers/src/HowWeAre/HowWeAre.jsx [hunk 5]
```diff
@@ -0,0 +1,69 @@
...
+                  </h3>
+                  <p 
+                  className=" text-white text-base font-semibold font-barlow  leading-[19.2px] px-4 mb-[24px] lg:leading-[21.6px]
+                  lg:px-0 lg:text-[18px]"
+                  >Somos un “startup” capacitados en brindar soluciones tecnológicas a tu empresa o negocio, tenemos experiencia en el desarrollo web, aplicaciones móviles y software con diseños pensados para el usuario. Nos destacamos por la fusión de creatividad y estrategias personalizadas para nuestros clientes, donde estaremos asistiéndote durante todo el proceso asegurando la calidad y garantía en nuestros servicios.
+                  </p>
+                </div>
+              </div>
+            </div>
+            <div className=" lg:h-[568px] grid grid-cols-1">
```
Comment (HamiltonG98) [inline apps/web/containers/src/HowWeAre/HowWeAre.jsx:35]: aqui igual, tenes un lg de primero, deberia ir de ultimo

**Review by HamiltonG98 — CHANGES_REQUESTED** [review]
Revisa el linter que estas ocupando, vi algunas cosas cosas que no estaban formateadas y no es codigo legible

File: apps/web/containers/src/HowWeAre/HowWeAre.jsx (hunk 4 above)
Comment (Dierish05) [reply apps/web/containers/src/HowWeAre/HowWeAre.jsx]: si, eso hacia, en algunos si tuve el error de dejarlos donde no era

**Review by HamiltonG98 — APPROVED** [review]

**Pull request merged**